src/app/my-own-custom-material.module.ts

```diff
@@ -0,0 +1,27 @@
+import { MatButtonModule, MatInputModule, MatFormFieldModule,
+  MatToolbarModule, MatTabsModule, MatTableModule, MatExpansionModule, MatSelectModule } from '@angular/material';
+import { NgModule } from '@angular/core';
+
+@NgModule({
+  imports: [
+    MatButtonModule,
+    MatInputModule,
+    MatFormFieldModule,
+    MatToolbarModule,
+    MatTabsModule,
+    MatTableModule,
+    MatExpansionModule,
+    MatSelectModule
+  ],
+  exports: [
+    MatButtonModule,
+    MatInputModule,
+    MatFormFieldModule,
+    MatToolbarModule,
+    MatTabsModule,
+    MatTableModule,
+    MatExpansionModule,
+    MatSelectModule
+  ],
+})
+export class MyOwnCustomMaterialModule { }
```

Comment:
create a folder inside the layout folder by the name of `custom-material` and put this file inside that.

Reply:
The top level components inside the `/app` should only be of the app component.